pkg/kubelet/kubelet.go

```diff
@@ -2569,6 +2566,17 @@ func (kl *Kubelet) setNodeStatus(node *api.Node) error {
 		}
 		oldNodeUnschedulable = node.Spec.Unschedulable
 	}
+
+	// NOTE(aaronlevy): NodeReady condition needs to be the last in the list of node conditions.
+	// This is due to an issue with version skewed kubelet and master components.
+	// ref: https://github.com/kubernetes/kubernetes/issues/16961
```

Comment:
In the release-1.1 branch, I don't think it would be dropped. This is a backport of a change that was made in master, and I could add a note there (although, it might be worthwhile just removing it from master altogether, as 1.2/1.0 version skew is not likely to be supported).